admin-dev/themes/default/scss/font.scss

```diff
@@ -33,3 +33,7 @@
   /* Support for IE. */
   font-feature-settings: "liga";
 }
+
+.d-none {
+  display: none !important;
```

Comment:
The `!important` is mandatory in this case?
I'm surprised this class didn't exist already in Bootstrap 3 🤷‍♂️ But yeah turns out it's been introduced in Bootstrap 4

Reply:
Well I m not sure how much important is needed but this is how it looks on Bootstrap 4, so I added the same part.